src/api/app/helpers/webui/user_helper.rb

```diff
@@ -25,9 +25,13 @@ def user_actions(user)
         mail_to(user.email) do
           tag.i(nil, class: 'fas fa-envelope text-secondary pe-1', title: 'Send Email to User')
         end,
-        link_to('#', title: 'Delete User', data: { 'bs-toggle': 'modal',
-                                                   'bs-target': '#delete-user-modal', 'user-login': user.login, action: user_path(user.login) }) do
-          tag.i(nil, class: 'fas fa-times-circle text-danger pe-1')
+        if user.state == 'deleted'
+          ''
+        else
```

Comment:
I think you can omit this slice and do just `if user.state != 'delete'`

Reply:
> I think you can omit this slice and do just `if user.state != 'delete'`

Yes, it works. I only had to invert the logic of the conditional, to pass RuboCop. Thanks!